fruit_classifier/preprocessing/preprocessing_utils.py

```diff
@@ -4,6 +4,45 @@
 from skimage.transform import resize
 from pathlib import Path
 from fruit_classifier.utils.file_utils import copytree
+import os
+
+
+def truncate_filenames(raw_dir):
+    # Assumes raw_dir is a directory that contains ONLY directories,
+    # in which all files/folders will be truncated if their total
+    # path length is more than 255 characters.
+
+    folder_list = os.listdir(raw_dir)
```

Comment:
Recall that the name `folder` is Windows specific. Use `directory` or `dir` to make it multi purpose

```suggestion
    dir_list = Path(raw_dir).glob('*')
```